databricks_cli/clusters/cli.py

```diff
@@ -157,17 +172,18 @@ def list_node_types_cli():
 @eat_exceptions
 def spark_versions_cli():
     """
-    List possible spark versions for a cluster.
+    Lists possible spark versions for a cluster.
```

Comment:
We should say `Apache Spark or Databricks Runtime versions`. Let's discuss this offline.